Source/src/importers/ImporterManager.cpp

```diff
@@ -8,149 +8,103 @@
 #include "MaterialImporter.h"
 #include "AnimationImporter.h"
 #include "FontImporter.h"
+#include "PrefabImporter.h"
 
 using namespace Hachiko;
 
 ImporterManager::ImporterManager()
 {
-    const auto mesh = new MeshImporter();
-    const auto model = new ModelImporter();
-    const auto texture = new TextureImporter();
-    const auto material = new MaterialImporter();
-    const auto animation = new AnimationImporter();
-    const auto font = new FontImporter();
-
-    importers.emplace(model->GetType(), model);
-    importers.emplace(mesh->GetType(), mesh);
-    importers.emplace(texture->GetType(), texture);
-    importers.emplace(material->GetType(), material);
-    importers.emplace(animation->GetType(), animation);
-    importers.emplace(font->GetType(), font);
+
+    // They exist both as assets and as resources
+    material = new MaterialImporter();
+    texture = new TextureImporter();
+    font = new FontImporter();
+    prefab = new PrefabImporter();
+
+    // It doesnt have its own resource type (we use prefabs)
+    model = new ModelImporter();
+
+    // They dont have their own asset type
+    animation = new AnimationImporter();
+    mesh = new MeshImporter();
```

Comment:
Maybe in the next iteration we can have these non-pointer as they are already created at the ctor.

Reply:
Doing it for this one owo

Reply:
love u uwu